records_mover/records/schema/schema/__init__.py

```diff
@@ -160,6 +160,13 @@ def from_fileobjs(fileobjs: List[IO[bytes]],
             fileobj.seek(0)
 
             df = purge_unnamed_unused_columns(df)
+            # TODO: Move into function
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker